ConverterViewController.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  ConverterViewController.swift
+//  anarmyofones
+//
+//  Created by Fabian Mariño on 4/4/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+
+// MARK: Local Constants
+
+// Define server side script URL
+let EXCHANGESURL = "https://api.fixer.io/latest?base=USD&symbols=GBP,EUR,JPY,BRL"
+let REQUESTMETHOD = "GET"
+let CURRENCIES = [ "UK": "GBP",
+                   "EUR": "EUR",
+                   "JAP": "JPY",
+                   "BRA": "BRL"
+                 ]
+let FIELDMAXLENGTH = 8
+let DEFAULTAMOUNT = 1.0
+
+class ConverterViewController: UIViewController, UITextFieldDelegate {
+    
+    var _dollarAmount: Double? = DEFAULTAMOUNT
+    var _exchangesDictionary:[String:Double]?
+    let _currencyFormatter : NSNumberFormatter = NSNumberFormatter()
+    var _requestedWasOk = false
+    
+    @IBOutlet weak var lblGBP: UILabel!
```

Comment:
Don't use "lbl" prefixes.  iOS / Swift convention is to use type suffixes.   "GBPLabel" or "PoundsLabel"